### DISCUSSION TITLE
Is GraphQL still supported?

### DISCUSSION THREAD

**Discussion by rod-stuchi** [mistweaverco/kulala.nvim#55] [Q&A] [answered]
because GraphQL exemples were removed from documentation

**Comment by gorillamoe** [comment]
Working on getting graphql support back into Kulala.

The examples got removed, when I released the complete rewrite of the parser.

Can you create an issue like "GraphQL Support is missing" or something like that, please?

**Reply by gorillamoe** [reply]
NVM, I created an issue here: https://github.com/mistweaverco/kulala.nvim/issues/58

**Reply by rod-stuchi** [reply]
Thank you!

**Reply by gorillamoe** [reply, marked as answer]
Re-Implemented in v2.3.0 -> https://kulala.mwco.app/docs/usage/graphql
